public/createmessage.php

```diff
@@ -27,7 +27,7 @@
 
 if ($messageContextID != -1) {
     $messageContextData = GetMessage($user, $messageContextID);
-    $messageContextTitle = "RE: " . $messageContextData['Title'];
+    $messageContextTitle = "RE: " . addcslashes($messageContextData['Title'],"'");
```

Comment:
The php-cs-fixer is failing because there isn't a space between parameters in the function call.